### DISCUSSION TITLE
What would be technology for Pimcore Backend UI

### DISCUSSION THREAD

**Discussion by anhttStartup** [pimcore/pimcore#15224] [General]
Hi, we have been working with Pimcore for few years; we love the Pimcore native Backend UI but it's not user-friendly with end-users. That's why we always have to build a customized Administrative Interface based on ReactJS or other JS template (which based on Bootstrap).

I see in the [Roadmap](https://pimcore.com/en/developers/roadmap) that Pimcore has plan to replace Sencha Ext JS; but does anyone know what should be the Technology / Frontend Framework would be chosen?

Best regard,

**Comment by SamyMP** [comment]
Hello,

It has been discussed a little bit [here](https://github.com/pimcore/pimcore/discussions/12795), nothing has been officially announced though. We are still waiting for news about it 🙂

**Reply by dpfaffenbauer** [reply]
AFAIK: still same as my comment back then: react or vue ;)
